src/Taxonomy_Command.php

```diff
@@ -148,12 +148,22 @@ protected function get_counts( $taxonomies ) {
 	public function list_( $args, $assoc_args ) {
 		$formatter = $this->get_formatter( $assoc_args );
 
+		// Check if it's strict mode or not.
+		$is_strict = ( isset( $assoc_args['strict'] ) && true === $assoc_args['strict'] ) ? true : false;
```

Comment:
There's a helper function called `get_flag_value` for retrieving flags like this. Providing a third argument of `false` means it defaults to `false`, i.e. `--no-strict`.

Also, by convention, we usually use the same name for the variable as we have for the flag: `--strict` => `$strict`.
```suggestion
		$strict = Utils\get_flag_value( $assoc_args, 'strict', false );
```